src/Tasks/Microsoft.NET.Build.Tasks/RunReadyToRunCompiler.cs

```diff
@@ -19,7 +20,7 @@ public class RunReadyToRunCompiler : ToolTask
         [Required]
         public ITaskItem CompilationEntry { get; set; }
         [Required]
-        public ITaskItem[] ImplementationAssemblies { get; set; }
+        public ITaskItem[] References { get; set; }
```

Comment:
I think the term `References` here could add ambiguities in the future, and someone could end up passing non-implementation assemblies as references to crossgen. I would like the names to reflect the notion of "implementation assemblies" used as crossgen references here if possible. Maybe something like `ImplementationAssemblyReferences`?

Reply:
I'll rename this. Just implementation assemblies was confusing me as to whether it was for referencing or compiling.